webpack/assets/javascripts/react_app/components/PF4/TableIndexPage/Table/TableIndexHooks.js

```diff
@@ -19,9 +19,10 @@ export const useTableIndexAPIResponse = ({
   apiUrl,
   apiOptions = {},
   defaultParams = {},
+  method = 'get',
 }) => {
   let response = useAPI(
-    replacementResponse ? null : 'get',
+    replacementResponse ? null : method,
     apiUrl.includes('include_permissions')
       ? apiUrl
       : `${apiUrl}?include_permissions=true`,
```

Comment:
Unrelated, but this assumes the API URL didn't use any GET parameters before. I'd think that isn't a safe assumption and think it should parse things.